Resources/Prototypes/Entities/Mobs/NPCs/carp.yml

```diff
@@ -88,6 +88,43 @@
   - type: TypingIndicator
     proto: guardian
 
+- type: entity
+  name: Lia
+  parent: MobCarpMagic
```

Comment:
if this is a ghost role inheriting from magicarps, you need to redefine the ghost role component and give it some custom text.

Reply:
magicarps dont have the GhostTakeoverAvailable comp, that's for the Salvage Ruleset carps (which I'm not basing off of)

![image](https://user-images.githubusercontent.com/67359748/179659042-f2c2c39c-4e55-47f5-8eed-d2fde3c3b21d.png)